lib/fastlane/plugin/firebase_app_distribution/actions/login.rb

```diff
@@ -0,0 +1,65 @@
+require 'googleauth'
+require 'googleauth/stores/file_token_store'
+require "google/apis/people_v1"
+# require "googleauth"
+# require "googleauth/stores/file_token_store"
+require "fileutils"
+module Fastlane
+  module Actions
+    module SharedValues
+      TEST_AUTH_CUSTOM_VALUE = :TEST_AUTH_CUSTOM_VALUE
+    end
+
+    class LoginAction < Action
+      def self.run(params)
+        client_id = Google::Auth::ClientId.new("563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com", "j9iVZfS8kkCEFUPaAeJV0sAi")
```

Comment:
I think it would be good for readability if these two strings were put into constant class variables: https://www.rubyguides.com/2017/07/ruby-constants/

It would help the reader identify what the two strings are, and would improve the readability of this line.